resolver/resolver.go

```diff
@@ -3517,6 +3522,14 @@ func (r *Resolver) trimPackage(pkg *Package, name string) string {
 	return strings.TrimPrefix(name, fmt.Sprintf("%s.", pkg.Name))
 }
 
+func newMessageBuilderFromMessage(message *Message) *source.MessageBuilder {
```

Comment:
In some cases, specifically when traversing the dependency graph and all the messages, the resolver does not visit the messages based on the proto definition, so we must build the location builder from the Message...